tools/bin/ci_integration_test.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e
+
+# runs integration and/or standard tests for an integration name
+
+connector="$1"
+all_integration_tests=$(./gradlew integrationTest --dry-run | grep 'integrationTest SKIPPED' | cut -d: -f 4)
+all_standard_python_tests=$(./gradlew standardSourceTestPython --dry-run | grep 'standardSourceTestPython SKIPPED' | cut -d: -f 4)
+
+if [[ "$connector" == "all" ]] ; then
+  echo "Running: ./gradlew --no-daemon --scan integrationTest standardSourceTestPython"
```

Comment:
```suggestion
  cmd="./gradlew --no-daemon --scan integrationTest standardSourceTestPython"
  echo "Running: $cmd"
  $cmd
```

Reply:
Just `$cmd` doesn't work, and we'd need to add evals.
```
# jrhizor in ~/code/airbyte on git:jrhizor/add-other-builds ● [15:01:23]
→ cmd="echo hi"

# jrhizor in ~/code/airbyte on git:jrhizor/add-other-builds ● [15:03:10]
→ $cmd
zsh: command not found: echo hi
```

Since this is temporary anyways, going to just merge as is and improve later.